R/collect.R

```diff
@@ -5,22 +5,42 @@
 #'  = TRUE` should have been used.
 #' @param summarize A logical; should metrics be summarized over resamples
 #' (`TRUE`) or return the values for each individual resample. Note that, if `x`
-#' is created by [last_fit()], `summarize` has not effect.
+#' is created by [last_fit()], `summarize` has no effect. For the other object
+#' types, the method of summarizing predictions is detailed below.
+#' @param parameters An optional tibble of tuning parameter values that can be
+#'  used to filter the predicted values before processing. This tibble should
+#'  only have columns for each tuning parameter identifier (e.g. `"my_param"`
+#'  if `tune("my_param")` was used).
```

Comment:
Can there be a more transparent example of how to use this? Either literally a whole entry in `@examples` (maybe switch from `fit_resamples()` to tune_grid()`?) or else change the documentation here to something like:
```
This tibble should only have columns for tuned parameters (e.g., if `my_param = tune()` was used with tuning values of 3, 5 and 10, a possible value is `parameters = tibble(my_param = 3).` 
```